README.md

```diff
@@ -43,3 +43,19 @@ Dangerzone can convert these types of document into safe PDFs:
 Dangerzone was inspired by [Qubes trusted PDF](https://blog.invisiblethings.org/2013/02/21/converting-untrusted-pdfs-into-trusted.html), but it works in non-Qubes operating systems. It uses containers as sandboxes instead of virtual machines (using Docker for macOS, Windows, and Debian/Ubuntu, and [podman](https://podman.io/) for Fedora).
 
 Set up a development environment by following [these instructions](/BUILD.md).
+
+## FAQ
+
+### "I get `invalid json returned from container` on MacOS Big Sur or newer (MacOS 11.x.x or higher)"
+
+You _may_ be attempting to convert a file in a directory to which Docker Desktop does not have access. Dangerzone for Mac requires Docker Desktop for conversion. Docker Desktop, in turn, requires permission from MacOS to access the directory in which your target file is located.
+
+To grant this permission:
+
+1. On MacOS 13, choose Apple menu > System Settings. On lower versions, choose System Preferences.
+2. Tap into Privacy & Security in the sidebar. (You may need to scroll down.)
+3. In the Privacy section, tap into Files & Folders. (Again, you may need to scroll down.)
```

Comment:
In a Mac system with enabled SIP and OS version 13.2.1, I can't find "Docker" under "Privacy & Security" -> "Files and Folders". Granted, it may be because Docker is not somehow triggering SIP in that system. For what is worth, Docker is installed from the official .dmg.